### PR TITLE
Update Skia version to 1.68.1

### LINKED ISSUE #2: Latest versions are missing
Jefflyfin is now using SkiaSharp 1.68.1 but the arm binaries are still in 1.68.0, I believe it's responsible for this bug: https://github.com/jellyfin/jellyfin/issues/2471

I'd be happy to help, is there some documentation how to update those?

### PR DESCRIPTION
**Pull request by jairbubbles**
Please note that the NuGet package still needs to be generated and pushed to nuget.org. This PR is only the easy part :-D

I guess we'll need to wait on @EraYaN 

Fixes #2

### CODE REVIEW

File: Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ TOOLCHAIN_ARM64=gcc-linaro-4.9.4-2017.01-x86_64_aarch64-linux-gnu
 
 set -e
 if [ ! -d "skia" ] ; then
-    git clone https://github.com/mono/skia.git -b v1.68.0 --depth 1 skia
+    git clone https://github.com/mono/skia.git -b v1.68.1 --depth 1 skia
```
Comment (mark-monteiro) [inline Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: The latest version is `v1.68.1.1` I think we should use that?

Comment (jairbubbles) [reply Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: Well jellyfin uses currently `1.68.1`, see:

https://github.com/jellyfin/jellyfin/blob/b3811a9498fe06b68693f4a269de902cdd7eb2a2/Jellyfin.Drawing.Skia/Jellyfin.Drawing.Skia.csproj#L15

So I feel like it's safer to perfectly match. But we could also upgrade everywhere to `1.68.1.1`.

Comment (mark-monteiro) [reply Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: Ah okay, yeah I see. Someone else would have to make that call I'm not familiar with this project, probably @EraYaN

Comment (EraYaN) [reply Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: Ooh building this was quite a pain. I need to dig a bit to see which builds are still required. (Upstream also added some I think)

Comment (jairbubbles) [reply Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: I'm currently compiling using the bash script but I'm using WSL on windows and it's taking hours... but so far so good.

Comment (jairbubbles) [reply Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: Well after hours... it finally failed:
```
[31/1043] compile ../../src/ports/SkFontConfigInterface_direct_factory.cpp
FAILED: obj/src/ports/fontmgr_fontconfig.SkFontConfigInterface_direct_factory.o
arm-linux-gnueabihf-g++ -MMD -MF obj/src/ports/fontmgr_fontconfig.SkFontConfigInterface_direct_factory.o.d -DNDEBUG -DSK_SAMPLES_FOR_X -DSK_GAMMA_APPLY_TO_A8 -DSK_ALLOW_STATIC_GLOBAL_INITIALIZERS=0 -DSK_ENABLE_DISCRETE_GPU -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -march=armv7-a -mfpu=neon -mthumb -O3 -fdata-sections -ffunction-sections -Wno-unused-parameter -DSKIA_C_DLL -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -c ../../src/ports/SkFontConfigInterface_direct_factory.cpp -o obj/src/ports/fontmgr_fontconfig.SkFontConfigInterface_direct_factory.o
In file included from ../../src/ports/SkFontConfigInterface_direct_factory.cpp:8:0:
../../src/ports/SkFontConfigInterface_direct.h:14:35: fatal error: fontconfig/fontconfig.h: No such file or directory
 #include <fontconfig/fontconfig.h>
                                   ^
```

Comment (jairbubbles) [reply Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: (seems related to https://github.com/mono/SkiaSharp/issues/633)

Comment (jairbubbles) [reply Jellyfin.SkiaSharp.NativeAssets.LinuxArm/cross-compile-libSkiaSharp.sh:7]: Ok well I finally made it to compile by downloading `libfontconfig` manually (both includes and libs for arm).

It depends on your disto but for Ubuntu I add to extract the .deb manually:
```
wget http://ports.ubuntu.com/pool/main/f/fontconfig/libfontconfig1-dev_2.11.94-0ubuntu1_armhf.deb
ar x libfontconfig1-dev_2.11.94-0ubuntu1_armhf.deb
tar xvf data.tar.xz
```

And copying files to `tools/arm-bcm2708/arm-linux-gnueabihf/arm-linux-gnueabihf/` folder.

@EraYaN  A few remarks:
- ~I'm not sure why at the end of the script you compile for x64 too~ (EDIT: read too fast, it's arm64)
- ~Would be great to run the script through a docker container to have a reproductible build~ (EDIT2: just saw the docker file in the repo 😫 )

**Comment by jairbubbles** [comment]
*Using docker to build was working fine but I first tried to use WSL which was way more complex and I ended up doing manually what was already done in the different scripts* 😞 

Anyway it's now compiling and I could generate the `.nupkg` properly.

Here it is (renamed as `.zip`): [Jellyfin.SkiaSharp.NativeAssets.LinuxArm.1.68.1.zip](https://github.com/jellyfin/jellyfin-skiasharp-native/files/4396882/Jellyfin.SkiaSharp.NativeAssets.LinuxArm.1.68.1.zip)

**Review by EraYaN — APPROVED** [review]

**Comment by jairbubbles** [comment]
@EraYaN can you push the package to NuGet ? (at least in beta so that we can test)

**Comment by manzajaro** [comment]
Hi,

I have the same issue #2 , How can I update jellyfin?

I have launched "compile-libSkiaSharp.sh" and "cross-compile-libSkiaSharp.sh" and both fail.

"./compile-libSkiaSharp.sh: line 13: /build/buildtools/linux64/gn: No such file or directory"
"./cross-compile-libSkiaSharp.sh: line 62: ./bin/gn: cannot execute binary file: Exec format error"

Distributor ID: Ubuntu
Description:    Ubuntu 18.04.4 LTS
Release:        18.04
Codename:       bionic
aarch64

Thanks

**Comment by jairbubbles** [comment]
@manzajaro I would recommend to compile using docker by running `build-in-docker.sh` but obviously you need to install docker on your machine before.

**Comment by EraYaN** [comment]
I have asked @anthonylavado to upload the nupkg (since I can't myself), should appear somewhere this week.

**Comment by manzajaro** [comment]
Will it be available in the next jellyfin update?

**Comment by jairbubbles** [comment]
> I have asked @anthonylavado to upload the nupkg (since I can't myself), should appear somewhere this week.

@EraYaN Cool, in the meatime I'll try to generate a local docker ARM image and make sure it does indeed fix the issue.

> Will it be available in the next jellyfin update?

I certainly hope so!

**Comment by EraYaN** [comment]
It's validating now, should be up on NuGet within 24 hours.

**Comment by Juansero29** [comment]
I strangely had this problem too on version 1.68.1 with Xamarin.Forms on Visual Studio 2019 yesterday. I was deploying on a physical iPhone SE. Cleaning obj/bin on the .iOS project, then rebuilding, and redeploying did the trick!

**Pull request merged**